docs/manage/_content.rst

```diff
@@ -178,6 +184,30 @@ If Kolibri is installed on an older or a low-resource device, you can try the fo
 On Linux and OSX you will need to enable the **Show hidden folders** option in order to view the ``.kolibri`` folder.    
 
 
+Update Content Channels
+-----------------------
+
+Public channels on Kolibri Studio are periodically updated from their original sources. Private and unlisted channels that you have on your local Kolibri device may also have changed since you first imported them. You can update your local channels from Kolibri Studio from online sources, from another updated device in your local network or from an external drive where the latest version of the channels has been :ref:`previously exported <export>`.
+
+.. warning:: Don't update channels used to make :ref:`lessons <manage_lessons>` and :ref:`quizzes <manage_quizzes>` while those are **still active and visible to learners**. The structure and location of some resources inside the channel may have changed in the updated version, which can render them unavailable and create loading issues. It is recommended to **schedule and perform channel updates only during periods when learners are not actively working on lessons and quizzes**.
```

Comment:
Saying "don't do this" seems a little bit strong. It's also somewhat unrelated to the other guidance about doing imports when people are not actively working with Kolibri.

Perhaps break it into two independent warnings? Could do something like:

> **Warning:** Use caution when updating channels that have been used to make currently-active :ref:`lessons <manage_lessons>` and :ref:`quizzes <manage_quizzes>`. The structure and location of resources inside the channel may have changed in the updated version, which can cause lessons and quizzes to not load correctly.
>
> **Warning:** It is recommended to schedule and perform channel updates only during periods of low activity on the server. Updating content while people are using the server may cause errors or performance issues.